upload/admin/view/template/sale/order_invoice.twig

```diff
@@ -14,7 +14,7 @@
 <div class="container">
   {% for order in orders %}
   <div style="page-break-after: always;">
-    <h1>{{ text_invoice }} #{{ order.order_id }}</h1>
+    <h1>{% if order.invoice_no %}{{ text_invoice }} #{{ order.invoice_no }}{% else %}{{ text_order }}{% endif %}</h1>
```

Comment:
No in every country the **#** sign is used!
If someone need it, better to do it via the text - otherwise it has to be deleted by an OCMod or Event!

Reply:
Not every country. If you want to use # for Order AND Invoice:
`<h1>{% if order.invoice_no %}{{ text_invoice }} #{{ order.invoice_no }}{% else %}{{ text_order }} #{{ order.order_id }}{% endif %}</h1>`